app/views/idv/shared/_verify.html.erb

```diff
@@ -4,84 +4,94 @@ locals:
   remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP
   step_url - generator for step URLs
 %>
-<% title t('titles.doc_auth.verify_info') %>
+<% title t('titles.idv.verify_info') %>
 
-<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.verify')) %>
+<%= render PageHeadingComponent.new.with_content(t('headings.verify')) %>
 <div class='margin-top-4 margin-bottom-2'>
   <div class="grid-row grid-gap grid-gap-2 padding-bottom-1">
-    <div class='grid-col-fill'>
+    <dl class="grid-col-fill margin-bottom-0">
```

Comment:
At some point we may want to consider creating a convenience CSS class for these sorts of term:value lists, so that we can avoid the margin utility here and the `display-inline` below.

Reply:
Makes sense. Could you link me to an existing CSS class so I can see how that's currently done in the codebase?

Reply:
> Makes sense. Could you link me to an existing CSS class so I can see how that's currently done in the codebase? Not planning to do it as part of this ticket but I'd like to know how for next time

That's a good question. The stylesheets are a bit scattered as far as patterns go.

I could see a few options:

- Treat it like a ["component"](https://github.com/18F/identity-idp/tree/main/app/assets/stylesheets/components) for description list, optionally of an "unstyled" variety
   - e.g. `.description-list` or `.term-list` or `.description-list.description-list--unstyled` in a new file `_description-list.scss` or `_term-list.scss`
   - Ideally these component stylesheets would map to a [Rails component](https://github.com/18F/identity-idp/tree/main/app/components) or [React component](https://github.com/18F/identity-idp/tree/main/app/javascript/packages/components), which we _could_ do, but I don't think is strictly necessary unless you think there's some value in it.
- Add a [utility](https://github.com/18F/identity-idp/tree/main/app/assets/stylesheets/utilities) class, maybe even to one of the existing (typography?)
   - I'm not a huge fan of the utility classes since they're a hodgepodge of miscellaneous things, but this feels like it fits as a miscellaneous thing